docs/source/locale/uk/LC_MESSAGES/tutorial.po

```diff
@@ -101,6 +101,19 @@ msgstr "Організатор може редагувати аукціон ті
 msgid "When this period ends 403 error will be returned on editing attempt:"
 msgstr "Після закінчення цього періоду на спробу редагування буде повернуто помилку 403: "
 
+msgid "Organizer can set *enquiryPeriod.endDate*. The difference between the given date and *tenderPeriod.endDate* should not be less than 5 working days."
+msgstr "Організатор має можливість встановити *enquiryPeriod.endDate*. Ця дата повинна бути не пізніша, ніж за 5 днів до *tenderPeriod.endDate*."
+
+msgid "If the duration between *enquiryPeriod.endDate* provided by Organizer and *tenderPeriod.endDate* is less than 5 days "
+"`422 Unprocessable Entity` response will be returned with error message "
+"'the difference between enquiryPeriod.endDate and tenderPeriod.endDate should not be less than 5 working days'."
+msgstr "Якщо *enquiryPeriod.endDate*, яку передав Організатор, буде ближчою, ніж 5 днів до *tenderPeriod.endDate*, "
+"згенерується помилка `422 Unprocessable Entity` та у JSON відповіді буде передано повідомлення "
+"'the difference between enquiryPeriod.endDate and tenderPeriod.endDate should not be less than 5 working days'."
+
+msgid "If Organizer didn't set *enquiryPeriod.endDate* it will be calculated automatically as *tenderPeriod.endDate* minus 5 days."
```

Comment:
`does not` instead of `didn't` (so that to have 1st type of a conditional sentence)
Mixed type is great but not as it's provided here

Reply:
Also, 5 `working` days